content/doc/book/pipeline/multibranch.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: section
+---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:imagesdir: /doc/book/resources/pipeline
+:hide-uri-scheme:
+:toc:
+
+= Multibranch Pipelines
+
+In the <<jenkinsfile#, previous section>> a `Jenkinsfile` which could be
+checked into source control was implemented. This section covers the concept of
+*Multibranch* Pipelines which build on the `Jenkinsfile` foundation to provide
```

Comment:
What other advantages does passive voice get us aside from sounding official-y? I feel it gains us sounding official at the cost of clarity.   


If we can get away from passive voice, I vote we do so.

Active voice tends to be shorter and clearer:
``` 
The <<jenkinsfile#, previous section>> showed a `Jenkinsfile` which could be
checked into source control.
```

If we want to use a pronoun, I'd go with "we" rather than "you":
```
In the <<jenkinsfile#, previous section>>, we implemented a Jenkinsfile which could be checked into source control.
```